lte/gateway/c/core/oai/tasks/mme_app/mme_config.c

```diff
@@ -73,45 +73,6 @@ static bool parse_bool(const char* str);
 
 struct mme_config_s mme_config = {.rw_lock = PTHREAD_RWLOCK_INITIALIZER, 0};
 
-//------------------------------------------------------------------------------
-int mme_config_find_mnc_length(const char mcc_digit1P, const char mcc_digit2P,
```

Comment:
Why is this function moved to common_utility_funs.cpp?

Reply:
This function is being used by s1ap and s6a tasks, so moved common utility function.